src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -91,6 +94,15 @@ public void Awake()
 
     public void Start()
     {
+        if (RGUtils.IsMobile())
+        {
+            SetMobileView();
+        }
+        else if (RGUtils.IsPortraitView())
```

Comment:
Here @addisonbgross , I think this should be an `if`

Reply:
I dropped [a comment below](https://github.com/Regression-Games/RGUnityBots/pull/362#issuecomment-2501965271) that will hopefully address this comment 🤞🏻